internal/config/config.go

```diff
@@ -82,8 +82,6 @@ type FolderConfiguration struct {
 	IgnoreDelete    bool                        `xml:"ignoreDelete" json:"ignoreDelete"`
 
 	Invalid string `xml:"-" json:"invalid"` // Set at runtime when there is an error, not saved
-
-	deviceIDs []protocol.DeviceID
```

Comment:
We can't cache this as that makes the struct not compare correctly after changing the device list. It's no big deal, we don't call `DeviceIDs()` (that uses this) in any performance sensitive spots.